### PR TITLE
[MPS] Add Inverse op.

### PR DESCRIPTION
**Pull request by kulinseth**

### CODE REVIEW

**Review by DenisVieriu97 — APPROVED** [review]
looks good

**Review by DenisVieriu97 — COMMENTED** [review]
The build still seems to be failing due to missing selector for `inverseOfTensor`

File: aten/src/ATen/native/mps/operations/Inverse.mm [hunk 1]
```diff
@@ -0,0 +1,86 @@
+#include <ATen/ATen.h>
+#include <ATen/native/mps/OperationUtils.h>
+#include <torch/library.h>
+#include <c10/util/Optional.h>
+
```
Comment (DenisVieriu97) [inline aten/src/ATen/native/mps/operations/Inverse.mm:5]: I think we should import MPSGraphVenturaOps.h here. I'll quickly try locally to check if the build is gone
```suggestion
aten/src/ATen/native/mps/MPSGraphVenturaOps.h
```

**Review by DenisVieriu97 — CHANGES_REQUESTED** [review]

**Review by DenisVieriu97 — APPROVED** [review]

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**